cmd/tools/docs_gen/templates/http.adoc.tmpl

```diff
@@ -9,6 +9,8 @@
      https://github.com/redpanda-data/connect/tree/main/cmd/tools/docs_gen/templates/http.adoc.tmpl
 ////
 
+// Copyright Redpanda Data, Inc
```

Comment:
```suggestion
// © 2024 Redpanda Data Inc.
```
@Jeffail, Could all instances be updated to our standard copyright notice? Thanks